### PR TITLE
Impose bounds on all dependencies

### PR DESCRIPTION
**Pull request by timholy**
Closes #91
Closes #92
Closes #93

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -14,13 +14,16 @@ PaddedViews = "5432bcbf-9aad-5242-b902-cca2824c8663"
...
-ColorTypes = ">= 0.8.0"
-FixedPointNumbers = ">= 0.6.1"
-MappedArrays = ">= 0.2"
-OffsetArrays = ">= 0.8"
-PaddedViews = ">= 0.4.1"
-Reexport = ">= 0.2"
-julia = ">= 1.0"
+Colors = "0.9"
+ColorTypes = "0.8"
+FixedPointNumbers = "0.6.1"
```
Comment (tlnagy) [inline Project.toml:19]: Is this specific patch release required?

Comment (timholy) [reply Project.toml:19]: You'll note the old one was `>= 0.6.1`. [IIUC](https://julialang.github.io/Pkg.jl/v1/compatibility/), this allows anything in the range `[0.6.1, 0.7)`.

Comment (zygmuntszpak) [reply Project.toml:19]: I interpreted the documentation to mean that in order to have the range `[0.6.1, 0.7)` you need to specify the caret, e.g. `^0.6.1`.

The documentation you linked to gives this example:
`PkgD = "^0.2.3" # [0.2.3, 0.3.0)`

Comment (tlnagy) [reply Project.toml:19]: Caret specifiers are the default though so I believe @timholy's correct here.

Comment (johnnychen94) [reply Project.toml:19]: FYI, this is also documented in [BlueStyle](https://github.com/invenia/BlueStyle#package-version-specifications) as a recommendation.

**Pull request merged**